packages/pages/src/common/src/template/hydration.ts

```diff
@@ -82,6 +82,13 @@ export const getHydrationTemplate = (
     `;
 };
 
+const makeAbsolute = (path: string): string => {
```

Comment:
path.resolve should turn this into an absolute path

Reply:
These are url paths not system dependent paths and url paths all use forward-slashes.

Reply:
You can path.resolve(convertToPosixPath(path)) too, but this works aswell